### PR TITLE
Cyberwatch integration

### PR DESCRIPTION
**Pull request by tarraschk**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues

Asked by common Cyberwatch and Palo Alto customers, no issue open but Cyberwatch has been in touch with Palo Alto Technology Partnerships team.

Cyberwatch is a French cybersecurity software vendor, specialized in Vulnerability and Compliance Management.

## Description

This Pull Request provides a Pack with an Integration for the Cyberwatch software.

This PR allows Cortex XSOAR to:
- fetch assets from Cyberwatch
- fetch CVE data from Cyberwatch
- fetch Security Issues data from Cyberwatch

## Must have
- [X] Tests
- [X] Documentation

### CODE REVIEW

**Comment by samuelFain** [comment]
Hi @tarraschk ,
Thank you for your contribution!
Your PR review is slightly delayed and we will review it as soon as possible, we apologize for the inconvenience. 
In the meantime, please make sure the registration form is filled.
I'm available for any of your questions here or on Slack.
Thanks again.

**Comment by samuelFain** [comment]
@tarraschk, 
Thank you for your contribution and great work :)
Please merge the latest changes from `contrib/tarraschk_cyberwatch` into this branch (`Update branch` button) so we can activate the Github workflows needed to verify the changes in this PR.
Please feel free to reach out to me with any questions - I'm available here or on slack.

**Comment by tarraschk** [comment]
@samuelFain thank you for your message, I just rebased the PR

File: Packs/Cyberwatch/pack_metadata.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    ],
+    "githubUser": [
+        "tarraschk",
+        "fwininger"
+    ],
+    "marketplaces": [
+        "xsoar",
+        "marketplacev2",
+        "xsoar_on_prem",
+        "xsoar_saas"
```
Comment (samuelFain) [inline Packs/Cyberwatch/pack_metadata.json:35]: Some of these tags are irrelevant.
If you wish for this integration to run on all XSOAR versions (including XSIAM and XSOAR 8), then these two tags cover all the cases:
```suggestion
    "marketplaces": [
        "xsoar",
        "marketplacev2",
```

File: Packs/Cyberwatch/Integrations/Cyberwatch/Cyberwatch.py [hunk 2]
```diff
@@ -0,0 +1,925 @@
...
+            auth=(access_key, secret_key),
+            proxy=proxy)
+
+        if command == 'test-module':
+            # This is the call made when pressing the integration Test button.
+            result = test_module(client)
+            return_results(result)
+        # elif command == 'fetch-incidents':
+            # first_fetch = params.get('fetch_time').strip()
+            # fetch_incidents(client, first_fetch)
```
Comment (samuelFain) [inline Packs/Cyberwatch/Integrations/Cyberwatch/Cyberwatch.py:915]: Please remove commented code.

**Review by samuelFain — COMMENTED** [review]
Hi @tarraschk, Everything looks great!
Kindly review my comments.
We should schedule a demo along with @edik24 to make sure all the integration commands are functioning as expected, after which we should be good to merge the PR.
You can reach my on slack or mail via this address: sfainberg@paloaltonetworks.com 
Thanks!

File: Packs/Cyberwatch/Integrations/Cyberwatch/Cyberwatch.py (hunk 2 above)
Comment (tarraschk) [reply Packs/Cyberwatch/Integrations/Cyberwatch/Cyberwatch.py]: 👍 done

File: Packs/Cyberwatch/pack_metadata.json (hunk 1 above)
Comment (tarraschk) [reply Packs/Cyberwatch/pack_metadata.json]: 👍 done

**Review by samuelFain — APPROVED** [review]

**Pull request merged**